package-lock.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Do we need this file? I'm presuming its to get it working locally but I don't see the links going locally?

Reply:
I believe you are correct, not sure if it needs to be removed. Most repos tend to keep there package-lock.json files?

Reply:
It's only useful if we plan on running it offline, that means that the resources for bootstrap are getting loaded locally. It might be interesting to try and get this working but if it's not used then I would remove it and create another branch for running bootstrap locally,

Reply:
It's only useful if we plan on running it offline, that means that the resources for bootstrap are getting loaded locally. It might be interesting to try and get this working but if it's not used then I would remove it and create another branch for running bootstrap locally,